xbmc/settings/Settings.cpp

```diff
@@ -361,6 +361,7 @@ const std::string CSettings::SETTING_AUDIOOUTPUT_VOLUMESTEPS = "audiooutput.volu
 const std::string CSettings::SETTING_INPUT_PERIPHERALS = "input.peripherals";
 const std::string CSettings::SETTING_INPUT_PERIPHERALLIBRARIES = "input.peripherallibraries";
 const std::string CSettings::SETTING_INPUT_ENABLEMOUSE = "input.enablemouse";
+const std::string CSettings::SETTING_INPUT_ENABLE_CONTROLLER = "input.enablejoystick";
```

Comment:
Would it make more sense to make this specific to where it's used rather than generally here?